tests/hipscat_import/margin_cache/test_margin_cache.py

```diff
@@ -37,6 +38,10 @@ def test_margin_cache_gen(small_sky_source_catalog, tmp_path, dask_client):
 
     assert len(data) == 13
 
+    assert all(data[PartitionInfo.METADATA_ORDER_COLUMN_NAME] == norder)
+    assert all(data[PartitionInfo.METADATA_PIXEL_COLUMN_NAME] == npix)
+    assert all(data[PartitionInfo.METADATA_DIR_COLUMN_NAME] == int(npix / 10000) * 10000)
+
     catalog = Dataset.read_from_hipscat(args.catalog_path)
```

Comment:
I noticed this issue in the first place because I wanted to change this line to try loading the catalog as a HealpixDataset, and confirm that the metadata is all structured properly, and the checks all failed. Can you change the check to confirm that we can now load the created catalog?

Reply:
That makes sense, I've updated it to `HealpixDataset`